index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="flex-container">
+      <header>
+        <a href="" class="white text-normal align-center roboto">Mudasir Sherwani</a>
+        <nav>
+            <i class="fa-solid fa-bars fa-2x white"></i>
+        </nav>
+      </header>
+    </div>
+    <section>
+      <div class="shining-star"></div>
+      <div class="developer-title align-center" >
+        <h1 class="developer-name">Mr. Mudasir</h1>
+        <h2 class="software-developer">Software Developer</h2>
+      </div>
+        <p class="banner-text-pera">I can help you build a product, feature or website Look through some of my 
+          work and experience! if you like what you see and have a project you need coded, don't hasitate to 
+          contact me.
+        </p>
+    </section>
+</body>
```

Comment:
- [ ] You need to finish the header section by adding the social links and other lists, kindly consider adding this to your page

![Screenshot_20220817_203712](https://user-images.githubusercontent.com/25227022/185227806-f60d8008-46c2-4b0c-96c7-7a43d5585bdf.png)